application/src/pages/Login/Login.test.tsx

```diff
@@ -0,0 +1,24 @@
+import { render } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import { Login } from './Login';
+import { store } from '../../bll/store';
+import i18n from '../../utils/i18next';
+
+const setUp = () => (
+  <BrowserRouter>
+    <Provider store={store}>
+      <Login />
+    </Provider>
+  </BrowserRouter>
+);
```

Comment:
You can create one common test util wrapper end connect it ti your tests instead of declaring it in each file

Reply:
i created custom render in folder test-utils, used where it possible